.github/workflows/wkdev-sdk.yml

```diff
@@ -10,14 +10,12 @@ defaults:
   run:
     shell: bash
 jobs:
-  build:
-    runs-on: self-hosted
+  build_amd64:
+    runs-on: [self-hosted==amd64]
     steps:
       - name: Set tag name
         run: |
-          if [ "${GITHUB_BASE_REF}" = 'main' ]; then
-            echo "WKDEV_SDK_TAG=latest" >> "${GITHUB_ENV}"
-          fi
+          echo "WKDEV_SDK_TAG=amd64" >> "${GITHUB_ENV}"
```

Comment:
This isn't correct, the tag had a purpose for different branches.

You could append the arch to the existing tag.